.gitmodules

```diff
@@ -100,3 +100,7 @@
 	path = 3rdparty/fusion/fusion
 	url = ../../xioTechnologies/Fusion.git
 	ignore = dirty
+[submodule "3rdparty/discord-rpc/discord-rpc"]
+	path = 3rdparty/discord-rpc/discord-rpc
+	url = ../../Vestrel/discord-rpc
```

Comment:
Why can't we use discord/discord-rpc ?

Reply:
There is build error in rapidjson and clang-format is being force enabled but it's config contains duplicated line which makes it fail